app/views/messages/index.html.erb

```diff
@@ -19,9 +19,11 @@
 <section class="message--create">
   <div class="grid">
     <div class="grid__item width-two-thirds shift-one-sixth">
-      <%= form_for @message, remote: true, :html => {:class => "sendbar form-width--searchbar"}, builder: GcfFormBuilder do |f| %>
-      <%= f.gcf_send_message_field :body, "Send", autofocus: true, placeholder: "Send a text message" %>
-      <input name="client_id" type="hidden" value="<%= @client.id %>">
+      <%= form_for @message, remote: true, :html => {class: 'sendbar form-width--searchbar' }, builder: GcfFormBuilder do |f| %>
```

Comment:
Feel free to remove this part of the commit from the PR, since it's not really related to the feature spec changes.